cita-executor/src/backlogs.rs

```diff
@@ -388,7 +387,7 @@ impl Backlogs {
     }
 
     pub fn completed_keys(&self) -> ::std::vec::Vec<&u64> {
-        self.completed.keys().sorted()
+        unimplemented!()
```

Comment:
Is this implementation related to db replacement?

Reply:
I'll have another try.

Reply:
I am ok.